src/code-snippet/code-snippet.component.ts

```diff
@@ -193,6 +192,14 @@ export class CodeSnippet {
 		}, this.feedbackTimeout);
 	}
 
+	ngAfterViewInit() {
+		setTimeout(() => {
+			if (this.code ? this.code.nativeElement.getBoundingClientRect().height > 255 : false && this.display === "multi") {
```

Comment:
This is likely a touch more readable

```suggestion
			if ((this.code && this.code.nativeElement.getBoundingClientRect().height > 255) && this.display === "multi") {
```